scripts/upgrade.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
Ok, can work on writing something in workspaces-tests. This was more for ad-hoc testing, where you could upgrade in testnet and play around with upgraded contract maybe can just remove to avoid introducing JavaScript tests